HealthMonitoring.AcceptanceTests/Scenarios/Selenium/Home_page.cs

```diff
@@ -86,5 +87,27 @@ public void Applying_url_filter_to_endpoints()
                 _ => Then_only_endpoints_with_chosen_parameters_should_be_shown()
                 );
         }
+
+        [Scenario]
+        public void Filters_should_apply_when_traveling_forward_and_backward_on_history()
+        {
+            _driver.Manage().Timeouts().ImplicitlyWait(TimeSpan.FromSeconds(1));
```

Comment:
Please hide the implementation details from the scenario.
It should be moved to either constructor (if applicable to all scenarios) or to some given/setup steps